docs/docs/linking-between-pages.md

```diff
@@ -8,23 +8,47 @@ This guide covers how to link between pages in a Gatsby site.
 
 The Gatsby `<Link />` component is for linking between pages within your site. For external links to pages not handled by your Gatsby site, use the regular HTML `<a>` tag.
 
-## Using the `<Link />` component
+## Using the `<Link />` component for internal links
 
 Here's an example of creating a link between two pages in a Gatsby site.
 
-Open a page component (e.g. `src/pages/index.js`) in your Gatsby site. Import the `<Link />` component from Gatsby. Add a `<Link />` component below the header, and give it a `to` property, with the value of `"/contact/"` for the pathname:
+Open a page component (e.g. `src/pages/index.js`) in your Gatsby site. Import the `Link` component from Gatsby, which makes it available in the component. Add a `<Link />` component below the header, and give it a `to` property with the value of `"/contact/"` for the pathname:
 
 ```jsx
 import React from "react"
 import { Link } from "gatsby"
 
 export default () => (
   <div>
-    <Link to="/page-2/">Page 2</Link>
+    <Link to="/contact/">Contact</Link>
   </div>
 )
 ```
 
+The above code will add a link to the contact page, automatically rendered in HTML as `<a href="/contact/">` but with added performance benefits.
+
+> **Note:** the value `"/"` for the `to` property will take users to the home page.
```

Comment:
I love that this anticipates a need a beginner would definitely run into, but might not be aware of 👏